test/Interpreter/generic_casts.swift

```diff
@@ -158,16 +158,6 @@ print(allToAll(type(of: C()), AnyObject.self)) // CHECK-NEXT: true
 // Bridging
 print(allToAll(0, AnyObject.self)) // CHECK-NEXT: true
 
-// This will get bridged using _SwiftValue.
```

Comment:
These need to be readded in, but I need some help figuring out how to address the NSCopying one — if it is enough to `import Foundation` on Linux so that Foundation can then extend `_SwiftValue` with `NSCopying`.

Reply:
(This will require making Linux's `_SwiftValue` `public` as a Foundation SPI.)